core/templates/dev/head/player/conversation_skin_directive.html

```diff
@@ -103,16 +103,17 @@
                 </div>
                 <div ng-if="!exploration.isInteractionInline(activeCard.stateName)"
                      style="opacity: 0.8;">
-		     <div ng-if ="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" 
-			     style="text-transform: uppercase;padding: 6px 12px;"> 
-				  <[exploration.getInteractionInstructions(activeCard.stateName)]>
-		     </div>
-		   <div ng-if="(!isCurrentSupplementalCardNonempty() || isScreenNarrowAndShowingTutorCard())">
-			<md-button class="instructions-button"
-			     ng-click="showSupplementalCardIfScreenIsNarrow()">
-				  <[exploration.getInteractionInstructions(activeCard.stateName)]>
-			          <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
-			</md-button>
+                  <div ng-if="isCurrentSupplementalCardNonempty() && !isScreenNarrowAndShowingTutorCard()" 
+                       style="text-transform: uppercase; padding: 6px 12px;"> 
+		     <[exploration.getInteractionInstructions(activeCard.stateName)]>
+                     <i class="material-icons md-18" style="position: relative; top: 3px;">&#xE5C8;</i>
+                  </div>
+		  <div ng-if="(!isCurrentSupplementalCardNonempty() || isScreenNarrowAndShowingTutorCard())">
+                    <md-button class="instructions-button"
+                               ng-click="showSupplementalCardIfScreenIsNarrow()">
```

Comment:
Can you bring the line 113 to 112 (all in one line),
`<md-button class="instructions-button" ng-click="showSupplementalCardIfScreenIsNarrow()">`

You can do the same with 104 and 105. Make sure to use spaces instead of tabs.